.s2i/bin/assemble

```diff
@@ -66,6 +66,20 @@ mkdir community_package/community/httpd-cfg
 mv .s2i/httpd-cfg/01-community.conf community_package/community/httpd-cfg
 mv 404-community.html community_package/community/404.html
 
+# Optionally install .htaccess files to gate access to staging content
+# Define a HTACCESS_DIRS environment variable with ':' delimited directories
+# Reference directory like "commercial_package/commercial/container-platform/3.10:community_package/community/latest" for docs.openshift.com/container-platform/3.10 and docs.okd.io/latest
+if [[ ! -z "$HTACCESS_DIRS" ]]; then
+    echo "---> Creating .htaccess/.htpasswd files"
+    IFS=':'; directories=($HTACCESS_DIRS); unset IFS;
+    for dir in "${directories[@]}"; do
+        echo "--->     Adding .htpasswd protection to $dir"
+        echo -e 'AuthType Basic\nAuthName "Access to the stage docs"\nAuthUserFile /opt/app-root/src/.htpasswd\nRequire valid-user' > $dir/.htaccess
```

Comment:
@wgordon17 - overall looks good to me. I might not be following this part correctly though. The .htaccess file that is created here says that the location of the .htpasswd is /opt/app-root/src/.htpasswd, while the lines below seem to put it in another place altogether, separately for community and commercial package?

Reply:
@wgordon17 - ah I see. Sorry for my misunderstanding. Cool - let's merge this :)